docs/estimators/index.rst

```diff
@@ -104,6 +104,20 @@ n_sigma_ul        Number of sigma used for upper limit estimation
 ts_threshold_ul   TS threshold to define the use of an upper limit
 ================= =================================================
 
+A note on negative flux and upper limit values:
+
+.. note::
+
+    Gammapy allows for negative flux values and upper limits by default.
```

Comment:
Technically, this depends on the allowed parameter range, no? 
If you force, `amplitude` and  `norm` to be positive quantities the UL will be positive as well right?

Reply:
Indeed I forgot about the recent change, that allows taking into account the boundaries. I'll adapt the text accordingly.